LeftNavBarExample/Example/build.gradle

```diff
@@ -0,0 +1,17 @@
+apply plugin: 'com.android.application'
+android {
+    compileSdkVersion 20
```

Comment:
Because I had to split the original PR in two.
I updated the compile sdk and build tools at the end.
It was supposed to be on a single PR